Source/Core/Core/HW/VideoInterface.cpp

```diff
@@ -53,11 +51,19 @@ static UVIBorderBlankRegister    m_BorderHBlank;
 
 u32 TargetRefreshRate = 0;
 
-static u32 TicksPerFrame = 0;
-static u32 s_lineCount = 0;
-static u32 s_upperFieldBegin = 0;
-static u32 s_lowerFieldBegin = 0;
-static int fields = 1;
+static u32 s_clock_freqs[2] =
+{
+	27000000UL,
+	54000000UL,
+};
+
+static u64 s_ticks_last_line_start = 0;
```

Comment:
I guess some comments are needed to clarify what these variables are intended to store.